.github/workflows/backend-ci.yml

```
@@ -0,0 +1,94 @@
+name: Backend CI
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'backend/**'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'backend/**'
```

```suggestion
  push:
    branches:
      - main
  pull_request:
```

このCIをStatus Checkに追加した場合、`/backend`に変更がない場合job自体が実行されません。そうすると、`/backend`の変更がない場合にはStatus Checkをパスしなくなってしまうので、[https://github.com/dorny/paths-filter](dorny/paths-filter)などを使用してジョブ単位で実行条件を指定するようにした方が良いと思われます。ジョブの`if:`で実行条件を指定した場合は、一度job自体は実行した判定になるので、Status Checkをパスするようになります。

**参考文献：**

- [GitHub Actions による Renovate の安全自動マージ (4. pull_request 用の GitHub Actions Workflow を 1 つにまとめる) | Zenn.dev](https://zenn.dev/shunsuke_suzuki/articles/renovate-auto-merge-github-actions#4.-pull_request-%E7%94%A8%E3%81%AE-github-actions-workflow-%E3%82%92-1-%E3%81%A4%E3%81%AB%E3%81%BE%E3%81%A8%E3%82%81%E3%82%8B)
- [モノレポで不要なGithub Actions実行を最小にしつつマージブロック機能(マージ前必須チェックステータス)を使う方法の検討 | Zenn.dev](https://zenn.dev/bigwheel/articles/05accc6323de18)